pkg/api/types.go

```diff
@@ -271,10 +271,10 @@ type KubernetesAddon struct {
 	Data       string                    `json:"data,omitempty"`
 }
 
-// IsEnabled returns if the addon is explicitly enabled, or the user-provided default if non explicitly enabled
-func (a *KubernetesAddon) IsEnabled(ifNil bool) bool {
+// IsEnabled returns true if the addon is enabled
+func (a *KubernetesAddon) IsEnabled() bool {
 	if a.Enabled == nil {
```

Comment:
this can be `return to.Bool(a.Enabled)`

Reply:
Yes, though I have a slight preference for this as it's more explicit and maintainable (for devs who don't know what to.Bool does)